### PR TITLE
Added alastair frontend

### PR DESCRIPTION
**Pull request by blacksph3re**
I created the frontend code for alastair, but only made visible to those holding the use:alastair permission, as the whole thing is still pretty much in alpha state

### CODE REVIEW

File: client/menu.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
       {
         "name": "oms.circles.list",
         "label": "Circles",
-        "icon": "fa-circle",
+        "icon": "fa-alastairalastaircircle",
```
Comment (blacksph3re) [inline client/menu.json]: oops

**Comment by serge1peshcoff** [comment]
@blacksph3re can you do a few things before merging?
1) resolve the merge conflicts (it's most likely caused by me adding discounts)
2) update the commits names to match the conventional changelog format (`feat(alastair): something` or `fix(alastair): something)`) so it'd appear in changelog
3) add `npm version minor` after that, it'd bump a version and will generate a changelog based on commits

would be lovely to have

**Comment by serge1peshcoff** [comment]
also, for `package-lock.json` you can just remove it, then remove node_modules, then `npm install` and after that commit the new file, easier than figuring out what is wrong where :D

**Comment by serge1peshcoff** [comment]
and one more thing I forgot, can you run `npm run lint` and check if everything matches the ESLint code style? I've no idea why it doesn't run Travis checks on it but still

**Comment by serge1peshcoff** [comment]
Superseded by #146.

**Pull request closed without merging**